Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -64,6 +64,9 @@ open class ALKConversationViewModel: NSObject, Localizable {
     // MARK: - Outputs
 
     open var isFirstTime = true
+    var timer = Timer()
+    var messagePosition = 0
```

Comment:
In this ViewModel we've a list of messages, so which message's position is it? Can you rename it based on that?

Reply:
@mukeshthawani  updated